plugins/modules/wafv2_web_acl.py

```diff
@@ -392,6 +469,13 @@ def main():
     purge_rules = module.params.get("purge_rules")
     check_mode = module.check_mode
 
+    custom_response_bodies = {}
+    for name, body in module.params.get("custom_response_bodies").items():
+        custom_response_bodies[name] = snake_dict_to_camel_dict(body, capitalize_first=True)
+
+    if custom_response_bodies:
+        module.require_botocore_at_least('1.21.0', reason='to set custom response bodies')
+
```

Comment:
```suggestion
    if module.params.get("custom_response_bodies"):
        module.require_botocore_at_least('1.21.0', reason='to set custom response bodies')

        custom_response_bodies = {}
        for custom_name, body in module.params.get("custom_response_bodies").items():
            custom_response_bodies[custom_name] = snake_dict_to_camel_dict(body, capitalize_first=True)

```

Works with the removed default value from above.

Furthermore, Integration test is failing, because the `name` var was overwritten by the `for` loop.  
That's why it was created a wafv2 with the name `too_many_requests` instead of `"{{ web_acl_name }}"`